fdbclient/ClientLogEvents.h

```diff
@@ -44,19 +44,28 @@ namespace FdbClientLogEvents {
 	};
 
 	struct Event {
-		Event(EventType t, double ts) : type(t), startTs(ts) { }
+		Event(EventType t, double ts, const Optional<Standalone<StringRef>> &dc) : type(t), startTs(ts){
+			if (dc.present())
+				dcId = dc.get(); 
+		 }
 		Event() { }
 
-		template <typename Ar>	Ar& serialize(Ar &ar) { return serializer(ar, type, startTs); }
+		template <typename Ar>	Ar& serialize(Ar &ar) { 
+			if (ar.protocolVersion().version() >= (uint64_t) 0x0FDB00B063010001LL) {
```

Comment:
I know that in most places, we now try to avoid using the protocol version string explicitly and instead use aliases defined in flow/ProtocolVersion.h. The aliases are all based on features, though, so I'm not sure if it makes sense to use them here or not. I'm ok to keep this is as is, but feel free to change it if you decide you want to.

Reply:
I also saw the new way of using protocol version and wasn't sure if this should be used for this PR. But i reasoned that it will go away when moving to flat buffers and hence didn't use it.